scripts/prepdocs.ps1

```diff
@@ -8,5 +8,6 @@ foreach ($line in $output) {
 
 Write-Host "Environment variables set."
 
-pip install -r ./scripts/requirements.txt
+pip install -r ./scripts/requirements.txt --quiet
+# Add if having Azure authentication issues: --searchkey $env:AZURE_SEARCH_KEY  --storagekey $env:AZURE_STORAGE_KEY
```

Comment:
add comments about setting `AZURE_STORAGE_KEY` and `AZURE_SEARCH_KEY` as system environment variables.  Instead of adding them to `.env` file.  That would fix the issue on previous lines where the padding is lost, and you don't need to add the padding back.